cmd/up/up.go

```diff
@@ -539,6 +541,8 @@ func (up *upContext) start() error {
 		HasDeploySection: (up.Manifest.IsV2 &&
 			up.Manifest.Deploy != nil &&
 			(len(up.Manifest.Deploy.Commands) > 0 || up.Manifest.Deploy.ComposeSection.ComposesInfo != nil)),
+		RemoteDependencies: len(up.Manifest.Dependencies.GetRemoteDependencies()),
```

Comment:
I mean, are we able to know that some deploy data (remote and local sizes) comes from an up command? I know that we track it separately It may be useful to know that some data of the deploy command are the same as those of the up command (because up calls deploy) in order not to form conjectures like "for the up command the dependencies section has been used x times and for the deploy command y". Sometimes the deploy tracking comes from the up command. If I am not entirely clear we can discuss this in private.